src/SignalR/clients/csharp/Http.Connections.Client/src/HttpConnectionFactory.cs

```diff
@@ -86,23 +87,22 @@ internal static HttpConnectionOptions ShallowCopyHttpConnectionOptions(HttpConne
             {
                 HttpMessageHandlerFactory = options.HttpMessageHandlerFactory,
                 Headers = options.Headers,
-                Cookies = options.Cookies,
                 Url = options.Url,
                 Transports = options.Transports,
                 SkipNegotiation = options.SkipNegotiation,
                 AccessTokenProvider = options.AccessTokenProvider,
                 CloseTimeout = options.CloseTimeout,
-                Credentials = options.Credentials,
-                Proxy = options.Proxy,
-                UseDefaultCredentials = options.UseDefaultCredentials,
                 DefaultTransferFormat = options.DefaultTransferFormat,
                 WebSocketConfiguration = options.WebSocketConfiguration,
             };
 
-            // WASM doesn't support Crypto APIs and our setter throws if you try to assign null
-            if (options.ClientCertificates != null)
```

Comment:
>and our setter throws if you try to assign null

Reply:
Ping.

Reply:
The property is initialized as part of the HttpConnectionOptions ctor in non-browser environments and it's never allowed to be null after that.